PublicationsRouter/v2/api/API.md

```diff
@@ -76,24 +76,41 @@ If you have publicly hosted content (e.g. splash pages, full-text web pages, or
 
 ### Validation Endpoint
 
-The Validation API allows you to test that your data feed to the system will be successful.
+The Validation API allows you to test that your data feed to the system will be successful. The request must take the form:
+
+    POST /validate?api_key=<api_key>
+    Content-Type: application/json
+    
+    [Incoming Notification JSON]
 
 You must have "Publisher account" (as opposed to an "Institution repository account") to access this endpoint.
 
 #### Metadata-only request
 
 If you are sending only the notification JSON, the request must take the form:
 
-    POST /validate?api_key=<api_key>
+    POST /notification?api_key=<api_key>
```

Comment:
This is wrong here. But I think some changes are needed in the Notification Endpoint section below (where this can be introduced).